src/configure.ts

```diff
@@ -65,15 +73,60 @@ export async function jestPreviewConfigure(
     // }
   });
 
-  if (options.publicFolder) {
+  if (publicFolder) {
     createCacheFolderIfNeeded();
     fs.writeFileSync(
       path.join(CACHE_FOLDER, 'cache-public.config'),
-      options.publicFolder,
+      publicFolder,
       {
         encoding: 'utf-8',
         flag: 'w',
       },
     );
   }
 }
+
+function autoRunPreview() {
+  const originalIt = it;
+  const itWithPreview: jest.It = (name, callback, timeout) => {
+    let callbackWithPreview = undefined as jest.ProvidesCallback | undefined;
```

Comment:
Consider writing this as `let callbackWithPreview: jest.ProvidesCallback | undefined;`

Reply:
Thank you 👂 script guru @thanhsonng